ddl/column.go

```diff
@@ -127,6 +127,7 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 			job.State = model.JobStateCancelled
 			return ver, errors.Trace(err)
 		}
+		log.Infof("[ddl] add column, run DDL job %s, column info %#v, offset %d", job, columnInfo, offset)
```

Comment:
log it after finished add index?

Reply:
Or add log at start and finished?

Reply:
@winkyao 
Here is adding the column.
Why log it after finishing the operation?
@coocood 
We haven't this information in the start or finish DDL. Here is start `add column`.